src/data/set/countable.lean

```diff
@@ -27,10 +28,17 @@ constructive analogue of countability. (For the most part, theorems about
 -/
 protected def countable (s : set α) : Prop := nonempty (encodable s)
```

Comment:
Yes, changing the definition will break much more code. Actually, I think that we should hide it behind an `inductive` as we do with `set.finite` to avoid `unfreezingI` here and there, then use `set.countable_coe_iff` as the public API.